components/portal/widgets/partials/widget-card.html

```diff
@@ -21,15 +21,20 @@
 <md-card class="widget-frame" id="widget-id-{{ widget.nodeId }}" aria-label="{{ widget.title }} widget">
 
   <!-- MAINTENANCE MODE OVERLAY -->
-  <div class="overlay__maintenance-mode" ng-if="widget.lifecycleState === 'MAINTENANCE'">
-    <div class="maintenance-content">
+  <div class="overlay__widget-mode" ng-if="widget.lifecycleState === 'MAINTENANCE'">
```

Comment:
Why the change in class names?

Reply:
If we do end up using this for more than just maintenance mode, for example, using it for a widget message, just trying to make sure that the naming is tied to one purpose.